tdrs-frontend/docker-compose.dev.yml

```diff
@@ -0,0 +1,9 @@
+# Dev environment version of Docker compose, using Dockerfile.dev instead, and remapping to the nginx port
+version: "3"
+
+services:
+  tdp-frontend:
+    build:
+      dockerfile: Dockerfile.dev
+    ports:
+      - 3000:80
```

Comment:
When extending compose files, docker-compose will concatenate `ports` lists instead of overriding any, which will lead to conflicting port mappings if we include this in the base `docker-compose.yml`.

Reply:
Good catch!